Makefile

```diff
@@ -15,18 +15,20 @@
 # Set ENV to 'YELP' if FQDN ends in '.yelpcorp.com'
 # Otherwise, set ENV to the FQDN
 ifeq ($(findstring .yelpcorp.com,$(shell hostname -f)), .yelpcorp.com)
```

Comment:
What about rely on an env variable to decide whatever use internal or public pypi?
I'm thinking to something similar to ``YELP=any-thing make ...`` to use internal pypi

It could reduce a little the hacks and will be fully operative on Linux and OS X